client/operation_event_skill_data.go

```diff
@@ -0,0 +1,55 @@
+package client
```

Comment:
rename this file to `event_skill_data.go` please. :)

Reply:
Ok. Should I go and introduce a new interface for events as well? currently i'm deriving from operation. I don't know what else this entails though.

Reply:
Creating a new 'event' next to operation doesn't seem to be that easy. Specially this creates a problem in decode.go in decodeEvent() as it returns an Object with the interface operation. This gets then handed down to the router which also expects an operation interface. My suggestion would be that i rename the file for now, indicating that we will have multiple events in the future, but keep it an 'operation' internally for now.

Reply:
Agreed. :)